index.js

```diff
@@ -122,8 +122,8 @@ exports.createServer = function (opts) {
       ogr = ogr2ogr(json)
     }
 
-    if (req.body.fileName) {
-      ogr.options(['-nln', req.body.fileName])
+    if (req.body.outputName) {
```

Comment:
Renaming `fileName` to `outputName` seems like a breaking change, can we avoid that?

Reply:
'output Name' is as currently listed in the documentation ('fileName' does not exist). Perhaps it's better to correct it in the documentation? and maybe, just in case, still checking both values to avoid breaking in each scenario?

Reply:
oh, didn't realize that confusion existed.  we can leave this change as is then